### PR TITLE
form validation sucess

### PR DESCRIPTION
**Pull request by Varsani2520**
@punyakrit ,

in contact page form validation sucessfully done .

#193

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -2,7 +2,7 @@ GOOGLE_CLIENT_ID
 GOOGLE_CLIENT_SECRET 
 NEXTAUTH_URL 
 NEXTAUTH_SECRET 
-MONGODB_URI
+MONGODB_URI="mongodb+srv://Rni:Rni@cluster0.avoq4cu.mongodb.net/?retryWrites=true&w=majority&appName=Cluster0"
```
Comment (punyakrit) [inline .env.example:5]: your env creds are open change this

**Comment by Varsani2520** [comment]
@punyakrit ,i solved env issue now review it and if you want any chages then tell me

**Comment by Varsani2520** [comment]
@punyakrit ,please merged it and also give responsed if you want to any edit

**Comment by punyakrit** [comment]
this is a huge pr need time to review as a sololy manage repo

**Review by punyakrit — APPROVED** [review]

**Comment by Varsani2520** [comment]
i understood @punyakrit

**Pull request merged**